deployment/scripts/sonarqube.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+echo "SonarQube Scan"
+echo "Directory: $PWD"
+echo "JAVA:      $JAVA_HOME"
+
+mkdir $PWD/sonarqube/
+mkdir $PWD/sonarqube/download/
+mkdir $PWD/sonarqube/extract/
+mkdir $PWD/sonarqube/certs/
+mkdir $PWD/sonarqube/store/
+
+curl -o $PWD/sonarqube/certs/RH-IT-Root-CA.crt --insecure $RH_IT_ROOT_CA_CERT_URL
+
+$JAVA_HOME/bin/keytool \
+  -keystore /$PWD/sonarqube/store/RH-IT-Root-CA.keystore \
+  -import \
+  -alias RH-IT-Root-CA \
+  -file /$PWD/sonarqube/certs/RH-IT-Root-CA.crt \
+  -storepass redhat \
+  -noprompt
+
+export SONAR_SCANNER_OPTS="-Djavax.net.ssl.trustStore=$PWD/sonarqube/store/RH-IT-Root-CA.keystore -Djavax.net.ssl.trustStorePassword=redhat"
+
+
+export SONAR_SCANNER_OS="linux"
+if [[ "$OSTYPE" == "darwin"* ]]; then
+    export SONAR_SCANNER_OS="macosx"
+fi
+
+export SONAR_SCANNER_CLI_VERSION="4.6.2.2472"
+export SONAR_SCANNER_DOWNLOAD_NAME="sonar-scanner-cli-$SONAR_SCANNER_CLI_VERSION-$SONAR_SCANNER_OS"
+export SONAR_SCANNER_NAME="sonar-scanner-$SONAR_SCANNER_CLI_VERSION-$SONAR_SCANNER_OS"
+
+curl -o $PWD/sonarqube/download/$SONAR_SCANNER_DOWNLOAD_NAME.zip --insecure https://binaries.sonarsource.com/Distribution/sonar-scanner-cli/$SONAR_SCANNER_DOWNLOAD_NAME.zip
```

Comment:
Maybe left over `--insecure`? We're downloading a binary file from a foreign domain, we should verify the certificate is valid 🙂

Reply:
👍